custom/icds_reports/static/icds_reports/mobile/css/app.less

```diff
@@ -326,3 +329,40 @@
 .alert-maintenance .close {
   margin-left: 3px;
 }
+
+.text-underline {
+  text-decoration: underline;
+}
+
+.sort-popup {
+  width: 280px;
```

Comment:
nit: should this be `max-width`?

Reply:
I have checked it with bigger headings, but it didnot overflow.. So I left it that way. But I will change it to max-width anyway. Thanks for pointing out